.gitignore

```diff
@@ -77,7 +77,7 @@ Pipfile.lock
 CMakeCache.txt
 cmake_install.cmake
 CMakeFiles/
-compile_commands.json
+compile_commands*.json
```

Comment:
This is intentional as I ending creating multiple symbolic links with `compile_commands1.json` and `compile_commands2.json` to link with the `compile_commands.json` for `embedded` and `gtest`. Again, this is mostly for my personal use but I think it doesn't obfuscate anything and should be fine.